contrib/dropwizard/src/main/java/io/opencensus/contrib/dropwizard/DropWizardMetrics.java

```diff
@@ -68,9 +70,23 @@
    * @since 0.17
    */
   public DropWizardMetrics(List<com.codahale.metrics.MetricRegistry> metricRegistryList) {
+    this(metricRegistryList, MetricFilter.ALL);
+  }
+
+  /**
+   * Hook the Dropwizard registry into the OpenCensus registry.
+   *
+   * @param metricRegistryList a list of {@link com.codahale.metrics.MetricRegistry}s.
+   * @param metricFilter a filter to choose which metric to export
+   * @since 0.19
+   */
+  public DropWizardMetrics(
+      List<com.codahale.metrics.MetricRegistry> metricRegistryList, MetricFilter metricFilter) {
     Utils.checkListElementNotNull(
         Utils.checkNotNull(metricRegistryList, "metricRegistryList"), "metricRegistry");
+    Utils.checkNotNull(metricFilter, "metricFilter");
     this.metricRegistryList = metricRegistryList;
+    this.metricFilter = metricFilter;
```

Comment:
Combine line 87 and 89:
this.metricFilter = Utils.checkNotNull(metricFilter, "metricFilter");